conditional/templates/new_year.html

```diff
@@ -0,0 +1,59 @@
+{% extends "nav.html" %}
+{% block title %}
+New Year Guide
+{% endblock %}
+{% block body %}
+<div class="container main">
+    <div id="new-welcome">
+        <h3 class="page-title">Welcome!</h3>
+        <p>Is it that time of year already? Before we get started, please <strong>be careful and ready everything before continuing</strong>. Some actions taken on the next few steps can be detrimental if followed in the middle of the year. Once you are ready to proceed, click 'Begin' below.</p>
+        <a href="#" data-module="newYear" data-step="welcome" class="btn btn-primary btn-new-next center-block"><span class="glyphicon glyphicon-ok"></span> Begin</a>
+    </div>
+    <div id="new-clear" style="display: none;">
```

Comment:
Looks like you added a `.hidden` class, please use that instead of inline style here.